python/llm/dev/benchmark/all-in-one/run-igpu.sh

```diff
@@ -0,0 +1,10 @@
+source /opt/intel/oneapi/setvars.sh
+ 
+export USE_XETLA=OFF
+export SYCL_CACHE_PERSISTENT=1
+KERNEL_VERSION=$(uname -r)
+if [[ $KERNEL_VERSION != *"6.5"* ]]; then
+    export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```

Comment:
I am not sure whether this `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1` is useful for igpu on Linux. @MeouSker77 What do you think? :)

Reply:
not sure too